dspace-api/src/test/data/dspaceFolder/config/local.cfg

```diff
@@ -201,6 +201,8 @@ featured.service.teitok.description = A web-based platform for viewing, creating
 
 
 ##### Shibboleth #####
+# Turn off the discofeed, it is allowed by default
+shibboleth.discofeed.allowed = true
```

Comment:
what

Reply:
It must be allowed for integration tests because some of them could fail